src/env/default.rs

```diff
@@ -279,6 +279,16 @@ impl FileSystem for DefaultFileSystem {
         std::fs::remove_file(path)
     }
 
+    fn rename<P: AsRef<Path>>(&self, src_path: P, dst_path: P, keep_data: bool) -> IoResult<()> {
+        if keep_data {
+            std::fs::rename(src_path, dst_path)
+        } else {
+            self.delete(src_path)?;
```

Comment:
No no no, this logic is test only (I suggest to put it in `ObfuscatedFS`), `DefaultFileSystem` is for production use.